Changes Kubectl --version command's output format (#32515)
kubernetes/website: content/en/docs/tasks/tools/install-kubectl-linux.md

   ```bash
   kubectl version --client
   ```

>    Or use this for detailed view of version: [Some More About Output Options](https://kubernetes.io/docs/reference/kubectl/overview/#output-options)

Reviewer: [This PR as-is]
![image](https://user-images.githubusercontent.com/46767780/160281414-eb8c269e-d019-47c4-8d15-ce08331e5cd4.png)

[My suggestion]
```suggestion
   Or use this for detailed view of version ([available output options](/docs/reference/kubectl/#output-options)):
```
(Please note that I updated the link URL too.)

Author: So adding to all the 3 👍🏼

   ```

>    Or use this for detailed view of version:

Reviewer: Preview: https://deploy-preview-32515--kubernetes-io-main-staging.netlify.app/docs/tasks/tools/install-kubectl-linux/#:~:text=Some%20More%20About%20Output%20Options

IMO, the indentation of newly added text is not appropriate.
![image](https://user-images.githubusercontent.com/46767780/160258919-434affad-08d9-4673-bac8-1a33237a6b0d.png)

How about this?
```suggestion
   Or use this for detailed view of version: ([More about output options](/docs/reference/kubectl/#output-options))
```
